drivers/gpio/gpio_mmio32.c

```diff
@@ -35,29 +35,15 @@ static int gpio_mmio32_config(struct device *dev, int access_op,
 	struct gpio_mmio32_context *context = dev->driver_data;
 	const struct gpio_mmio32_config *config = context->config;
 
-	if (flags & GPIO_INT) {
-		return -ENOTSUP;
-	}
-
-	if (access_op != GPIO_ACCESS_BY_PIN) {
-		return -ENOTSUP;
-	}
-
 	if ((config->mask & (1 << pin)) == 0) {
 		return -EINVAL; /* Pin not in our validity mask */
 	}
 
-	if (flags & ~(GPIO_DIR_MASK | GPIO_POL_MASK)) {
+	if (flags & ~(GPIO_INPUT | GPIO_OUTPUT | GPIO_ACTIVE_LOW)) {
```

Comment:
We may want to include here `GPIO_OUTPUT_INIT_LOW`, `GPIO_OUTPUT_INIT_HIGH` since this flags are compulsory / every driver should support them.

Reply:
Done